lib/validators.js

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+
+/**
+ * Checks that key bytes have a type of Uint8Array and a specific length.
+ *
+ * @throws {TypeError|SyntaxError} - Throws a Type or Syntax error.
+ *
+ * @param {object} options - Options to use.
+ * @param {Uint8Array} options.bytes - The bytes being checked.
+ * @param {number} [options.expectedLength=32] - The expected bytes length.
+ * @param {string} [options.code] - An optional code for the error.
+ *
+ * @returns {undefined} Returns on success throws on error.
+ */
+export const checkKeyBytes = ({bytes, expectedLength = 32, code}) => {
```

Comment:
Use `function` not arrow function. Probably better to call this `assert` as well since it throws.

```suggestion
export function assertKeyBytes({bytes, expectedLength = 32, code}) {
```